transfers/amclient.py

```diff
@@ -1,224 +1,45 @@
-        args=(AM_API_KEY,),
-        opts=(AM_USER_NAME, AM_URL, OUTPUT_MODE)
-    ),
-    SubCommand(
-        name='completed-ingests',
-        help='Print all completed ingests.',
-        args=(AM_API_KEY,),
-        opts=(AM_USER_NAME, AM_URL, OUTPUT_MODE)
-    ),
-    SubCommand(
-        name='unapproved-transfers',
-        help='Print all unapproved transfers.',
-        args=(AM_API_KEY,),
-        opts=(AM_USER_NAME, AM_URL, OUTPUT_MODE)
-    ),
-    SubCommand(
-        name='transferables',
-        help='Print all transferable entities in the Storage Service.',
-        args=(SS_API_KEY, TRANSFER_SOURCE),
-        opts=(SS_USER_NAME, SS_URL, TRANSFER_PATH, OUTPUT_MODE)
-    ),
-    SubCommand(
-        name='aips',
-        help='Print all AIPs in the Storage Service.',
-        args=(SS_API_KEY,),
-        opts=(SS_USER_NAME, SS_URL, OUTPUT_MODE)
-    ),
-    SubCommand(
-        name='dips',
-        help='Print all DIPs in the Storage Service.',
-        args=(SS_API_KEY,),
-        opts=(SS_USER_NAME, SS_URL, OUTPUT_MODE)
-    ),
-    SubCommand(
-        name='aips2dips',
-        help='Print all AIPs in the Storage Service along with their corresponding DIPs.',
-        args=(SS_API_KEY,),
-        opts=(SS_USER_NAME, SS_URL, OUTPUT_MODE)
-    ),
-    SubCommand(
-        name='aip2dips',
-        help='Print the AIP with AIP_UUID along with its corresponding DIP(s).',
-        args=(AIP_UUID, SS_API_KEY),
-        opts=(SS_USER_NAME, SS_URL, OUTPUT_MODE)
-    ),
-    SubCommand(
-        name='download-dip',
-        help='Download the DIP with DIP_UUID.',
-        args=(DIP_UUID, SS_API_KEY),
-        opts=(SS_USER_NAME, SS_URL, DIRECTORY, OUTPUT_MODE)
-    )
-)
-
-
-def get_parser():
-    """Parse arguments according to the ``SUBCOMMANDS`` configuration. Return
-    an argparse ``Namespace`` instance representing the parsed arguments.
-    """
-    parser = argparse.ArgumentParser(
-        description='Archivematica Client',
-        formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument(
-        '--log-file', metavar='FILE', help='logfile', default=DEFAULT_LOGFILE)
-    parser.add_argument(
-        '--log-level', choices=['ERROR', 'WARNING', 'INFO', 'DEBUG'],
-        default='INFO', help='Set the debugging output level.')
-    subparsers = parser.add_subparsers(help='sub-command help',
-                                       dest='subcommand')
-    for subcommand in SUBCOMMANDS:
-        subparser = subparsers.add_parser(subcommand.name,
-                                          help=subcommand.help)
-        for arg in subcommand.args:
-            subparser.add_argument(
-                arg.name, help=arg.help, type=arg.type)
-        for opt in subcommand.opts:
-            subparser.add_argument(
-                '--' + opt.name, metavar=opt.metavar, help=opt.help,
-                default=opt.default, type=opt.type)
-    return parser
+import urllib3
```

Comment:
I think that you need to add urllib3 to the requirements/base.txt file. urllib3 became a dependency of requests but only as of a certain version (I don't know exactly which). Because our requests requirement is vague, it's possible to have requests installed but not urllib3. The other option would be to pin down a more specific (higher) requests version in our requirements.